### ISSUE TITLE
Saving before crop changes are locked in produces unexpected results

### ISSUE THREAD

**Issue by marcelgruber** [MattKetmo/darkroomjs#66]
The image will not appear to the user as they desired, and the Base64 image will be even more wrong.

Is there a way we can disable save until the user accepts or denies crop mode?

**Comment by Venorcis** [comment]
We have the same issue. If you save before you 'accept the crop', you actually save the image with the crop overlay on it. I guess the save button should be gone/disabled whilst cropping.

**Comment by marcelgruber** [comment]
Exactly. Or conversely, in the save event you could check if the crop plugin is currently in use. The question is how.

**Comment by ziplizard** [comment]
If I could recommend one enhancement, it would be exactly what Venorcis has suggested above. The save icon simply needs to be disabled/hidden while cropping. I'm currently facing a UI challenge where users click #save before clicking #done. This dilemma can be resolved very simply by disabling or hiding the #save button before/during cropping. I would be verify interested in this update. Thanks.
